samples/json.js

```diff
@@ -0,0 +1,17 @@
+import http from "k6/http";
+import { check } from "k6";
+
+export default function() {
+    // Send a JSON encoded POST request
+    let body = JSON.stringify({ key: "value" });
+    let r = http.post("http://httpbin.org/post", body, { headers: { "Content-Type": "application/json" }});
+
+    // Use JSON.parse to deserialize the JSON (instead of using the r.json() method)
```

Comment:
is Response.json() still available?  I think I got an error when I tried to use it very recently